draft-ion.adoc

```diff
@@ -315,9 +315,9 @@ no members, an Ion parser MUST NOT identify the member as a _Meta Object_.
 If a member name is not equal to the case-sensitive octet sequence `meta`, an Ion parser MUST NOT identify a member as
 a _Meta Object_.
 
-The member names within an identified _Meta Object_ MUST be unique; Ion parsers MUST either reject _Meta Object_
+The member names within an identified _Meta Object_ SHOULD be unique; Ion parsers MUST either reject _Meta Object_
```

Comment:
validation:

- section 2.2 of http://www.ietf.org/rfc/rfc4627.txt, referenced in http://www.ecma-international.org/ecma-262/5.1/#sec-15.12, uses SHOULD be unique
- conditions follow this statement for how non-unique values can still be parsed